app/commands/git/sync_track.rb

```diff
@@ -46,11 +46,15 @@ def concepts
 
     def concept_exercises
       config_concept_exercises.map do |exercise|
+        next unless exercise[:uuid] # TODO: decide if we want to allow null as the uuid
```

Comment:
```suggestion
        next unless exercise[:uuid].present? # TODO: decide if we want to allow null as the uuid
```

Guard against empty strings with `present?`